docs/quickstart.md

```diff
@@ -38,14 +38,20 @@ All further instructions assumes you are in the code directory eg. `ocp4-upi-pow
 
 Update the [var.tfvars](../var.tfvars) based on your environment. Description of the variables are available in the following [link](./var.tfvars-doc.md).
 
+For sensitive variables that should not be saved on disk, set these environment variables;
+```
+$ export IBMCLOUD_API_KEY=xxxxxxxxxxxxxxx
+$ export RHEL_SUBS_USERNAME=xxxxxxxxxxxxxxx
+$ export RHEL_SUBS_PASSWORD=xxxxxxxxxxxxxxx
+```
 
 ## Start Install
 
 Run the following commands from within the directory.
 
 ```
 $ terraform init
-$ terraform apply -var-file var.tfvars -parallelism=3
+$ terraform apply -var-file var.tfvars -parallelism=3 -var ibmcloud_api_key="$IBMCLOUD_API_KEY" -var rhel_subscription_username="$RHEL_SUBS_USERNAME" -var rhel_subscription_password="$RHEL_SUBS_PASSWORD"
```

Comment:
Provide both the invocations and you can mention like
"If using environment variables for sensitive data then use the following command"
...

Reply:
Yep, will be in the next push.